pkgs/development/python-modules/minio/default.nix

```diff
@@ -1,19 +1,37 @@
 { lib, buildPythonPackage, isPy3k, fetchPypi
-, urllib3, future, python-dateutil , pytz, faker, mock, nose }:
+, configparser
+, faker
+, future
+, mock
+, nose
+, python-dateutil
+, pytz
+, pytest
+, urllib3
+}:
 
 buildPythonPackage rec {
   pname = "minio";
   version = "5.0.10";
+  disabled = !isPy3k;
 
   src = fetchPypi {
     inherit pname version;
     sha256 = "6ecb7637a35f806733e9d112eacfa599a58d7c3d4698fda2b5c86fff5d34b417";
   };
 
-  disabled = !isPy3k;
+  propagatedBuildInputs = [
+    configparser
+    future
+    python-dateutil
+    pytz
+    urllib3
+  ];
 
-  checkInputs = [ faker mock nose ];
-  propagatedBuildInputs = [ urllib3 python-dateutil pytz future ];
+  checkInputs = [ faker mock nose pytest ];
```

Comment:
```suggestion
  checkInputs = [ faker mock nose pytestCheckHook ];
```

Reply:
was holding out to see if i had to do some gnarly --ignore or -k arguments, but it works fine with just the vanilla invocation, i can change it

Reply:
at least -k can be replaced with `disabledTest`.